src/main/java/com/google/code/or/binlog/impl/parser/AbstractRowEventParser.java

```diff
@@ -149,6 +135,19 @@ protected Row parseRow(XInputStream is, TableMapEvent tme, BitColumn usedColumns
 				final int blobLength = is.readInt(meta);
 				columns.add(BlobColumn.valueOf(is.readBytes(blobLength)));
 				break;
+			case MySQLConstants.TYPE_GEOMETRY:
+				final int geomLength = is.readInt(meta);
+				final int _unknown = is.readInt(4);

```

Comment:
in case we have mysterious crash, let's print out this value if it ever changes?